src/MaxText/sft/vllm_decode.py

```diff
@@ -0,0 +1,110 @@
+#  Copyright 2025 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+An example script to perform decoding using vLLM with a MaxText model.
+
+Example command:
+  python3 -m MaxText.sft.vllm_decode MaxText/configs/sft.yml \
+    model_name=llama3.1-8b tokenizer_path=meta-llama/Llama-3.1-8B-Instruct \
```

Comment:
For consistency, we should use both the `model_name` and the `tokenizer_path` as the `-Instruct` version